### PR TITLE
Use `fedora32` label for these jobs

### PR DESCRIPTION
**Pull request by deepshikhaaa**

### CODE REVIEW

File: build-gluster-org/jobs/32-bit-build-smoke.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 - job:
     name: gh_32-bit-build-smoke
-    node: fedora30
+    node: fedora32
```
Comment (mscherer) [inline build-gluster-org/jobs/32-bit-build-smoke.yml:3]: So now I am reading that, I wonder if Fedora 30 is not the last Fedora with a 32 bits build. Have the jobs been tested on fedora 32 ?

Comment (deepshikhaaa) [reply build-gluster-org/jobs/32-bit-build-smoke.yml:3]: I have not tested it yet. I can give it a try.

Comment (deepshikhaaa) [reply build-gluster-org/jobs/32-bit-build-smoke.yml:3]: https://build.gluster.org/view/Github%20jobs/job/gh_32-bit-build-smoke/1176/, it failed. It has a i386 mock config for F32 but the job is failing at installing gluster.

Comment (deepshikhaaa) [reply build-gluster-org/jobs/32-bit-build-smoke.yml:3]: It was missing libuuid and rpcgen packages.

**Pull request merged**